src/entities/card/organisms/card-preview.tsx

```diff
@@ -18,8 +18,8 @@ import {
 type CardType = 'item' | 'details';
 
 interface CardPreviewProps {
-  card: Card | null;
-  isCardInFavorite: boolean;
+  card?: Card;
```

Comment:
Ох, похоже поругаемся с Олегом)))

Но лан, пока норм

Reply:
пофикси тут тайп эрроры.
```suggestion
  card?: Card | null;
```